apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1/project.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "version": "0.1.0-*",
+  "description": "Google Compute Engine Metadata v1 client library",
```

Comment:
I'm saying that if it were a "regular" API we'd use a different description, but I think it's fine for now. I'm okay keeping the v1 in there if you prefer.